lib/features/composer/presentation/composer_controller.dart

```diff
@@ -578,169 +579,184 @@ class ComposerController extends BaseController
     }
   }
 
-  void _initEmail() {
+  Future<void> _initEmail() async {
     _isEmailBodyLoaded = false;
     final arguments = PlatformInfo.isWeb
       ? mailboxDashBoardController.composerArguments
       : Get.arguments;
-    if (arguments is ComposerArguments) {
-      composerArguments.value = arguments;
 
-      _initIdentities(arguments);
+    if (arguments is! ComposerArguments) return;
 
-      injectAutoCompleteBindings(
-        mailboxDashBoardController.sessionCurrent,
-        mailboxDashBoardController.accountId.value
+    if (arguments.emailActionType == EmailActionType.forward) {
+      final newEmailSubject = await sanitizeEmailSubject(
```

Comment:
Asynchronous calling here still safe for us in building composer view?

Reply:
I still prefer using synchronize calling `HtmlEcscape().convert(headerQuotedContent)`. Benefit:
- not touch to the mechanism of composer loading, especially long rich content, inline images, ...
- handle html escape for content of `reply`, `replyAll`, `forward`

Keep transformHtml with escapeTransformer handler for printEmail, and other stuffs.

Reply:
If using `TransformHtmlEmailContentInteractor` is still desired, prefer wrapping `TransformHtmlEmailContentInteractor` in another interactor for `subject` alone, and use `handleSuccessViewState` and `handleFailureViewState` to handle result, rather than blocking the `_initEmail` call with `async-await`

Reply:
done

Reply:
https://github.com/user-attachments/assets/7a45049b-ca1b-4f89-8290-d5a30c5d1c4b